docs/Dockerfile

```diff
@@ -5,7 +5,7 @@ MAINTAINER Mu Li <muli@cs.cmu.edu>
 # First, build MXNet binaries (ref mxnet/docker/cpu/Dockerfile)
 #
 
-RUN apt-get update && apt-get install -y build-essential git libopenblas-dev libopencv-dev
+RUN apt-get update && apt-get install -y build-essential git libopenblas-dev liblapack-dev libopencv-dev
```

Comment:
We can't break the build for ubuntu 14.04 users.

I see three possible options:
1. add USE_LAPACK in config.mk and turn it off by default
2. automatically detect if a lapack lib is available and turn on/off lapack accordingly.
3. Build our own lapack when make